addons/rose/addon/styles/rose/components/table/_table.scss

```diff
@@ -10,9 +10,23 @@
     text-align: left;
   }
 
+  .rose-table-row-key-value {
+    td:nth-child(2) {
```

Comment:
`Key/Value` styles defined with a nth-child. 
Assumption:
- `key/value` rows will have JUST two cells. One for the key and another for the value.
- The cell that contains the value can have 1 or more elements. In example, an input + a button. That's why I defined with `display: flex`. So if we add more elements they will work just fine.
- If new use cases of `key/value` are shown in designs, we might need to tweak it, but for the use cases we have now its fine.

Reply:
Actually the style applied it is just for the `value` cell. I do not encounter any issue with the `key` cell. That said, if in any use case the `key` cell gets too bigger I agree we can rely on `shrink` for it.

About the `value` cell, I used the special-casing mostly to apply `display: flex` so the items we add on that cell flow. But happy to consider other options.

Reply:
Update: after a zoom call we decide to forget about `display:flex` so we delete the `rose-table-row-key-value` style and approach. The path to follow is use 3 rows in these cases (key/value lookalike editing mode).